server/vulnerabilities/io/github.go

```diff
@@ -63,7 +65,30 @@ func (gh GitHubClient) Download(URL string) (string, error) {
 }
 
 // Bulletins returns a map of 'bulletin name' => 'download URL' of the bulletins stored as assets on Github.
-func (gh GitHubClient) Bulletins(ctx context.Context) (map[SecurityBulletinName]string, error) {
+func (gh GitHubClient) MSRCBulletins(ctx context.Context) (map[MetadataFileName]string, error) {
+	return gh.list(ctx, mSRCFilePrefix, NewMSRCMetadata)
+}
+
+func (gh GitHubClient) MacOfficeReleaseNotes(ctx context.Context) (MetadataFileName, string, error) {
```

Comment:
Maybe a comment to explain what this function does? 
Either put a comment or consider renaming to something clear like:
GetMacOfficeReleaseNotesList
GetMacOfficeReleaseNotesSomething